src/extension/wps/src/test/java/au/org/emii/aggregator/NetcdfAggregatorTest.java

```diff
@@ -206,22 +206,32 @@ public void testLatLonProjectionSpatialSubsetNoData() throws IOException, Aggreg
         assertNetcdfFilesEqual(resourcePath("au/org/emii/aggregator/projection-point-outside-expected.nc"), outputFile);
     }
 
-    @Test
-    public void testAggregationOverrides() throws IOException, AggregationException {
+    private void testAggregationOverrides(String resourcePath) throws IOException, AggregationException {
         LatLonRect bbox = new LatLonRect(new LatLonPointImmutable(-30.68, 97.82), new LatLonPointImmutable(-30.64, 97.86));
         CalendarDateRange timeRange = CalendarDateRange.of(CalendarDate.parseISOformat("gregorian", "2017-02-01T03:19:60"),
-            CalendarDate.parseISOformat("gregorian", "2017-02-02T03:19:60"));
+                CalendarDate.parseISOformat("gregorian", "2017-02-02T03:19:60"));
         AggregationOverrides overrides = AggregationOverridesReader.load(
-            resourcePath("au/org/emii/aggregator/template.xml"));
+                resourcePath(resourcePath));
 
         try (NetcdfAggregator netcdfAggregator = new NetcdfAggregator(
-            outputFile, overrides, bbox, null, timeRange
+                outputFile, overrides, bbox, null, timeRange
         )) {
             netcdfAggregator.add(resourcePath("au/org/emii/aggregator/srs-1.nc"));
             netcdfAggregator.add(resourcePath("au/org/emii/aggregator/srs-2.nc"));
         }
 
         assertNetcdfFilesEqual(resourcePath("au/org/emii/aggregator/overrides-expected.nc"), outputFile);
+
+    }
+
+    @Test
+    public void testAggregationOverridesWithAttributeDataType() throws IOException, AggregationException {
+        testAggregationOverrides("au/org/emii/aggregator/template.xml");
```

Comment:
I think keeping different template files will give us the flexibility to test different scenarios without making many changes to the tests.